src/abseil-cpp/preprocessed_builds.yaml.gen.py

```diff
@@ -166,7 +166,16 @@ def resolve_hdrs(files):
 
 
 def resolve_srcs(files):
-  return [ABSEIL_PATH + "/" + f for f in files if f.endswith(".cc")]
+  srcs = [ABSEIL_PATH + "/" + f for f in files if f.endswith(".cc")]
+  # Replace absl/random/internal/randen_hwaes.cc with
+  #         src/abseil-cpp/randen_hwaes_nohw.c
+  # to disable HWAES completely. (https://github.com/grpc/grpc/issues/26478)
+  try:
+    i = srcs.index(ABSEIL_PATH + "/absl/random/internal/randen_hwaes.cc")
+    srcs[i] = "src/abseil-cpp/randen_hwaes_nohw.cc"
```

Comment:
Another thing to investigate here is to:

- see whether  the necessary HW instructions (I think that's NEON for ARM64) are already available in the ARM64 architectures we support (we support aarch64 / ARM64 v8) - if they are, it should be fine to just build on ARM64 with neon instructions enabled and everything is going to work, without needing to resort to a hack

- alternatively, we could use InsecureBitGen from ABSL, which IMHO wouldn't require HW instructions to be present so even if we enable NEON at compile time, the instructions wouldn't be used at runtime anyway (which avoids the problems with some processors potentially not supporting them).

I think it make sense to thing about the two paths above before resorting to using a hack.

Reply:
I believe that NEON is supported by the all ARM architectures that gRPC supports. Challenge here is that gRPC needs to provide proper compilation option depending on the architecture. Otherwise it will throw an exception. So we need to update all build environments (makefile, php, python, ...) to have that thing. Given the fact that we don't need to have super-fast BitGen, it looks too much burden to maintain.

That's why I changed it to use InsecureBitGen instead. It's already super fast even without HW acceleration. But we still have this PR because it's about compilation. Although BitGen is not used by gRPC, its code needs to be compiled.